### DISCUSSION TITLE
Aggiornamento 1.5.5

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#550] [Richieste di aiuto]
Buongiorno,ho provato mille volte ad aggiornare il registro alla versione 1.5.5 e mi esce sempre il seguente errore:Forbidden
You don't have permission to access this resource.

Additionally, a 403 Forbidden error was encountered while trying to use an ErrorDocument to handle the request.

la cosa strana è che ho fatto tutto passo dopo passo seguendo questa guida che mi avete dato per lo stesso problema che si verificava alla versione precedente:E' un problema del server che non permette l'esecuzione di una procedura per un tempo prolungato.

Devi fare l'aggiornamento manuale: anzi tutto carica il file della versione 1.5.4 sul server e decomprimilo andando a sovrascrivere i file esistenti. In alternativa, decomprimi sul tuo computer il file della versione 1.5.4 e poi carica tutti i file con Filezilla. Questo secondo metodo è più lento, ma da' maggiori garanzie che non ci siano file mancanti.

Fatto questo, crea sulla cartella principale, il file ".gs-updating" che deve contenere le seguenti 3 righe:

version:1.5.4
build:0
token:codice
Per il token usa un codice a piacere.

A questo punto termina l'installazione andando alla pagina:
nome_sito/install/update.php?token=codice&step=2

In ogni caso allego file d'errore ,spero mi potiate aiutare al più presto.

[2023-07-18_00-40.log](https://github.com/iisgiua/giuaschool/files/12074265/2023-07-18_00-40.log)

**Comment by trinko** [comment]
Nel log non sono riportati messaggi di errore, forse non hai attivato il debug prima di generare l'errore. Assicurati che sia attivo il debug e vai alla pagina che genera l'errore per produrre un log completo.

Comunque, ho dovuto effettuare una piccola modifica alla versione rilasciata, per cui consiglio di riscaricare quella che trovi ora, anche se non credo sia questo il motivo dell'errore.

**Comment by trinko** [comment]
Scusa, non mi ero accorto che la sintassi del file da creare ".gs-updating" è sbagliata.l
Il file ".gs-updating" che deve contenere le seguenti 3 righe:
```
token="codice"
version="1.5.5"
build="0"
```
Per semplicità lo allego: decomprimilo e spostalo nella cartella principale del registro

[file.zip](https://github.com/iisgiua/giuaschool/files/12078451/file.zip)
